### PR TITLE
fix: Move files in /usr/etc to /etc

### PR DESCRIPTION
**Pull request by KyleGospo**
Adding files to /usr/etc is considered undefined behavior

### CODE REVIEW

File: system_files/shared/usr/lib/systemd/system/ublue-etc-merge.service
Comment (m2Giles) [inline system_files/shared/usr/lib/systemd/system/ublue-etc-merge.service:1]: Probably should remove this file. This is for the whole takeover your computer with a docker container

**Comment by castrojo** [comment]
Ok I've removed all mentions of the etc-merge service. Please doublecheck my work.

**Comment by castrojo** [comment]
This good to go or are we still sorting out that containers.json thing? (Does that even affect Bluefin since we're not using the rechunker?)

**Comment by p5** [comment]
> This good to go or are we still sorting out that containers.json thing? (Does that even affect Bluefin since we're not using the rechunker?)

@castrojo The containers.json thing is only affecting main and ucore, since this is where we add the container policy to the image.  The issue there is we want to move the policy from /usr/etc to /etc, but Fedora's containers package already has taken "ownership" of that file, and doesn't allow us to overwrite it from our own RPM. 
This has been reverted for now, but I don't think we have any issues here.

**Pull request merged**